### PR TITLE
DT-1085: 🔧 Add PECS Supplier to list of 2fa accounts

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

**Review by Mjwillis — APPROVED** [review]
LGTM

File: src/test/java/uk/gov/justice/digital/hmpps/oauth2server/integration/ClientLoginSpecification.kt [hunk 1]
```diff
@@ -26,6 +28,7 @@ import java.util.*
  * done automatically in circle configuration for automated builds, but needs enabling when running these tests.  By
  * default the dev profile doesn't have it enabled so that other clients can use this project without issues.
  */
+@ExtendWith(RemoteClientExtension::class)
 class ClientLoginSpecification : AbstractAuthSpecification() {
   private val clientBaseUrl = "http://localhost:8081/login"
```
Comment (petergphillips) [inline src/test/java/uk/gov/justice/digital/hmpps/oauth2server/integration/ClientLoginSpecification.kt:33]: so our remote client is set to port 8081, which used to succeed even though there is nothing listening on 8081.  Chromedriver 85 then causes the test to fail if chrome can't connect.

File: src/test/java/uk/gov/justice/digital/hmpps/oauth2server/resource/RemoteClientExtension.kt [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.digital.hmpps.oauth2server.resource
+
+import com.github.tomakehurst.wiremock.WireMockServer
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig
+import org.junit.jupiter.api.extension.AfterAllCallback
+import org.junit.jupiter.api.extension.BeforeAllCallback
+import org.junit.jupiter.api.extension.BeforeEachCallback
+import org.junit.jupiter.api.extension.ExtensionContext
+
+class RemoteClientExtension : BeforeAllCallback, AfterAllCallback, BeforeEachCallback {
```
Comment (petergphillips) [inline src/test/java/uk/gov/justice/digital/hmpps/oauth2server/resource/RemoteClientExtension.kt:10]: we therefore have to define a remote client wiremock instance listening on 8081

File: .gitignore [hunk 3]
```diff
@@ -20,9 +20,4 @@
...
 /out/
 
-### NetBeans ###
-/nbproject/private/
-/nbbuild/
-/dist/
-/nbdist/
-/.nb-gradle/
\ No newline at end of file
+dps-gradle-spring-boot-suppressions.xml
```
Comment (petergphillips) [inline .gitignore:23]: this is provided by the dps gradle spring boot project, so isn't required to be in our code base

**Review by JamieDraperUK — APPROVED** [review]
lgtm

**Pull request merged**